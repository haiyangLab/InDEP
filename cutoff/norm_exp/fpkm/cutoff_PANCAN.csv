,cutoff
0,0.6
