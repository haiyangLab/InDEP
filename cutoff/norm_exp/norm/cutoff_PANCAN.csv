,cutoff
0,0.7010416666666667
