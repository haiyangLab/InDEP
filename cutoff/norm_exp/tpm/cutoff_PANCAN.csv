,cutoff
0,0.5635416666666667
