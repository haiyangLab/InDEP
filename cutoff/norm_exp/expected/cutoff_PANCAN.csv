,cutoff
0,0.5989583333333334
